### ISSUE TITLE
Publish to npm

### ISSUE THREAD

**Issue by badsyntax** [seiyria/angular-bootstrap-slider#56]
As the title says.

**Comment by seiyria** [comment]
I don't really have a workflow that facilitates publishing in multiple places just yet (ie, grunt-bump or gulp-bump), so this is probably not going to happen anytime soon.

**Comment by tellijo** [comment]
+1

(I'm sure you've already seen this but you have a good workflow example there: http://ponyfoo.com/articles/my-first-gulp-adventure)

**Comment by seiyria** [comment]
I've got a good workflow I'd put in place if I did (see [this](https://github.com/seiyria/c/blob/master/gulpfile.js)), I just don't have time to work on this.

**Comment by ofairfoul** [comment]
+1

**Comment by windbender** [comment]
+1 ^ 3

:-)

**Comment by marie-dk** [comment]
+1

**Comment by madflow** [comment]
@seiyria Can we do anything to help?

**Comment by seiyria** [comment]
It's really just a matter of me making a workflow to help out, but this hasn't really been a priority. You can still install via npm with `npm install seiyria/angular-bootstrap-slider`.

**Comment by madflow** [comment]
Fair enough - if you come up with some tasks to help automating the release workflow - let us know ;)
